libs/dh/message-archive/feature-log-search/src/lib/dh-message-archive-log-search.component.ts

```diff
@@ -117,11 +119,26 @@ export class DhMessageArchiveLogSearchComponent implements OnDestroy {
         ? q.get('invocationId')
         : null;
     });
+  }
 
-    this.searchCriteria.dateTimeFrom =
-      this.initDateFrom().toISOString().split('.')[0] + 'Z';
-    this.searchCriteria.dateTimeTo =
-      this.initDateTo().toISOString().split('.')[0] + 'Z';
+  private buildRsmOptions() {
+    const entries = Object.entries(DocumentTypes);
+    entries.forEach((entry) => {
+      this.rsmFormFieldOptions.push({
+        value: entry[0],
+        displayValue: entry[1] + ' - ' + entry[0],
+      });
+    });
+  }
+
+  private buildProcessTypesOptions() {
+    const entries = Object.entries(ProcessTypes);
+    entries.forEach((entry) => {
+      this.processTypeFormFieldOptions.push({
+        value: entry[0],
+        displayValue: entry[0] + ' - ' + entry[1],
+      });
+    });
```

Comment:
suggestion: instead of modifying the `processTypeFormFieldOptions` property by pushing array items to it, consider building the array in the method and returning it. This way you keep the function "pure" and with less side-effects. So it could be something like:

```suggestion
  private buildProcessTypesOptions() {
    return Object.entries(ProcessTypes).map((entry) => {
      return {
        value: entry[0],
        displayValue: entry[0] + ' - ' + entry[1],
      };
    });
```